addons/stock/__manifest__.py

```diff
@@ -96,8 +96,6 @@
     'assets': {
         'web.report_assets_common': [
             # legacy reports (delete when all reports are converted)
-            'stock/static/src/legacy_web_report/utils.js',
-            'stock/static/src/legacy_web_report/report.js',
             'stock/static/src/legacy_web_report/report_backend.scss',
```

Comment:
You can delete all 3 lines + the comment about needing to delete the lines too if you want to keep it clean/less confusing. I'm not sure if it's also safe/worth deleting the files too since they will be orphans.

If you want to nitpick about the commit message, you can also refer to how https://github.com/odoo/odoo/pull/100741 and https://github.com/odoo/odoo/pull/100710 had already converted the legacy reports into OWL so the legacy stuff isn't needed anymore. They were removed in master from saas-16.2 onwards (https://github.com/odoo/odoo/pull/112664/) probably because there was no need to do it in stable at the time.

Reply:
In this PR, I only want to remove the JS, citing the fact that it's currently disabled. If you deem it safe to delete the CSS line or the asset files themselves, you can create a new PR for that, referring to the Owl conversion.